mysql-test/suite/rpl/t/optimized_replace_mixed_engine-master.opt

```diff
@@ -0,0 +1 @@
+--enable_blind_replace
```

Comment:
For command-line options general convention is to use dashes(`-`) instead of underscores(`_`)
```suggestion
--enable-blind-replace
```